apps/services/tests/test_models.py

```
@@ -0,0 +1,10 @@
+"""Model tests for services."""
+import pytest
+
+
+@pytest.mark.django_db
+def test_service_queryset_active_filters_inactive(service_factory, vendor_factory):
```

Ruff (0.14.0)

Unused function argument: `vendor_factory`

(ARG001)